paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -347,30 +432,33 @@ def wait_for_deployment(service, deploy_group, git_sha, soa_dir, timeout):
         raise NoInstancesFound
     paasta_print("Waiting for deployment of {0} for '{1}' complete..."
                  .format(git_sha, deploy_group))
-    for cluster in cluster_map.values():
-        cluster['deployed'] = []
-    total_instances = sum([len(v["instances"]) for v in cluster_map.values()])
+
+    total_instances = 0
+    for cluster in cluster_map:
+        cluster_map[cluster]['deployed'] = 0
+        cluster_map[cluster]['queue'] = Queue()
+        cluster_map[cluster]['num_instances'] = len(cluster_map[cluster]['instances'])
```

Comment:
No difference indeed.
The idea was to create a short reference for 'len(cluster_map[cluster]['instances'])' but something went wrong.
I'll revert it.